eng/config/PublishData.json

```diff
@@ -78,7 +78,6 @@
       "Microsoft.CodeAnalysis.ExternalAccess.OmniSharp": "arcade",
       "Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.CSharp": "arcade",
       "Microsoft.CodeAnalysis.ExternalAccess.AspNetCore": "vs-impl",
-      "Microsoft.CodeAnalysis.ExternalAccess.Razor": "vs-impl",
```

Comment:
Don't remove this, we need it for servicing branches

Reply:
oh interesting. Does the publish data only come from one branch?

Reply:
yep, only main.  we kind of want to change it, but haven't yet